pkg/deploy/clusterrolebinding.go

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) 2012-2019 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package deploy
+
+import (
+	"context"
+	"github.com/sirupsen/logrus"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	runtimeClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func SyncClusterRoleBindingToCluster(
+	deployContext *DeployContext,
+	name string,
+	serviceAccountName string,
+	clusterRoleName string) (*rbac.ClusterRoleBinding, error) {
+
+	specCRB, err := getSpecClusterRoleBinding(deployContext, name, serviceAccountName, clusterRoleName)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterRB, err := getClusterRoleBiding(specCRB.Name, deployContext.ClusterAPI.Client)
+	if err != nil {
+		return nil, err
+	}
+
+	if clusterRB == nil {
+		logrus.Infof("Creating a new object: %s, name %s", specCRB.Kind, specCRB.Name)
+		err := deployContext.ClusterAPI.Client.Create(context.TODO(), specCRB)
+		return nil, err
+	}
+
```

Comment:
Let's check if `clusterRB` and `specCRB` are equal, like here.
https://github.com/eclipse/che-operator/blob/master/pkg/deploy/pvc.go#L71-L80

Reply:
ok. Should I change return value to ProvisioningStatus?

Reply:
No, that's ok